webapp/config/packages/nelmio_api_doc.yaml

```diff
@@ -252,7 +252,7 @@ nelmio_api_doc:
                         roles:
                             type: array
                             items:
-                                type: sring
+                                type: string
```

Comment:
Why did this not fail in the first place?

Reply:
I suspect because you can also have custom types OpenAPI is not aware of. This typo was the reason that I started playing around with the tool.